functional-tests/package.json

```diff
@@ -8,19 +8,22 @@
     "start": "pm2 update && pm2 start www",
     "stop": "pm2 delete www",
     "test-ci": "npm run start && npm run kakunin -- --parallel 4 && npm run stop",
-    "test": "npm run start && npm run kakunin && npm run stop"
+    "test": "npm run start && npm run kakunin && npm run stop",
+    "logs": "pm2 logs www"
   },
   "author": "",
   "dependencies": {
-    "body-parser": "^1.18.3",
+    "body-parser": "^1.19.0",
```

Comment:
Switch to express.json() , express.urlencoded(...)

Reply:
I think we don't use it anywhere.